src/util/util_watchdog.c

```diff
@@ -78,9 +75,7 @@ static void watchdog_handler(int sig)
 
     /* if a pre-defined number of ticks passed by kills itself */
     if (__sync_add_and_fetch(&watchdog_ctx.ticks, 1) > WATCHDOG_MAX_TICKS) {
-        DEBUG(SSSDBG_FATAL_FAILURE,
-              "Watchdog timer overflow, killing process!\n");
-        orderly_shutdown(1);
```

Comment:
One difference between the orderly shutdown and just _exit() is that orderly shutdown also kills the child processes in the process group with:
{{{
kill(-getpgrp(), SIGTERM);
}}}

I wonder if it's safe to just do exit or if we risk leaving zombies around if a child process was also running in the meantime?